.github/workflows/docs-deploy.yml

```
@@ -172,8 +185,17 @@ jobs:
       - name: Generate deployment summary
         if: always()
         run: |
```

actionlint (1.7.8)

property "job-start" is not defined in object type {deployment: {conclusion: string; outcome: string; outputs: {string => string}}}

(expression)